src/Controls/src/SourceGen/Controls.SourceGen-net6.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <AssemblyName>Microsoft.Maui.Controls.SourceGen</AssemblyName>
+    <RootNamespace>Microsoft.Maui.Controls.SourceGen</RootNamespace>
+    <PackageId>Microsoft.Maui.Controls.SourceGen</PackageId>
+    <NoWarn>$(NoWarn);NU5128;</NoWarn>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <None Include="$(OutputPath)\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(OutputPath)\Microsoft.Maui.Controls.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+    <None Include="$(OutputPath)\Microsoft.Maui.Controls.Xaml.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
```

Comment:
all the dependencies of the source generator need to be available in the analyser folder, otherwise the analyser can't start